types/stylelint/index.d.ts

```diff
@@ -850,6 +850,15 @@ declare namespace stylelint {
 		fix?: () => void | undefined | never;
 	};
 
+	/** @internal */
+	export type ProblemWithNormalizedPositions = Problem & {
+		line: number;
+		start: Position;
+		end: Position;
+		index: undefined;
+		endIndex: undefined;
+	};
```

Comment:
```suggestion
	export type ProblemWithNormalizedPositions = Omit<Problem, 'index' | 'endIndex'> &
                                                     Required<Pick<Problem, 'line' | 'start' | 'end'>>;
```

Reply:
Cool!

I didn't know about `Omit` and `Required`

Reply:
https://www.typescriptlang.org/docs/handbook/utility-types.html